pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "relate-courseware"
+version = "2016.1"
+description = "RELATE courseware"
+readme = "README.rst"
+repository = "https://github.com/inducer/relate"
+documentation = "https://documen.tician.de/relate/"
+authors = ["Andreas Kloeckner <inform@tiker.net>"]
+license = "MIT"
+packages = [
+    { include = "course" },
+    { include = "accounts" },
+    { include = "relate" },
+    # { include = "bin" },
+]
+
+[tool.poetry.scripts]
+relate = "relate.bin.relate:main"
+
+[tool.poetry.dependencies]
```

Comment:
This is makes `requirements.txt` redundant IMO. We shouldn't (try to) maintain both.